### PR TITLE
SO-720b - write email address if no name provided

### PR DESCRIPTION
**Pull request by adohall**
These changes in SO-720 weren't applied somehow. Still required.

### CODE REVIEW

File: app/templates/profile-builder.hbs [hunk 1]
```diff
@@ -19,9 +19,11 @@
       <h4>
         {{#if this.model.seller.name}}
           {{this.model.seller.name}}
+        {{else if auth.user.email}}
```
Comment (armansa) [inline app/templates/profile-builder.hbs:22]: This condition always returns true! This is not the supplier contact name, it's the email user has used to login.

File: app/templates/seller-account.hbs [hunk 2]
```diff
@@ -19,6 +19,8 @@
       <h4>
         {{#if this.model.seller.name}}
           {{this.model.seller.name}}
+        {{else if auth.user.email}}
```
Comment (armansa) [inline app/templates/seller-account.hbs:22]: On account seller always has name

File: app/templates/seller-dashboard.hbs [hunk 3]
```diff
@@ -18,6 +18,8 @@
       <h4>
         {{#if this.model.seller.name}}
           {{this.model.seller.name}}
+        {{else if auth.user.email}}
+          {{auth.user.email}}
```
Comment (armansa) [inline app/templates/seller-dashboard.hbs:22]: On dashboard seller name may night be available but auth.user.email is not supplier email it's user's email

Comment (armansa) [reply app/templates/seller-dashboard.hbs:22]: I would just print something like "Supplier Dashboard" here

Comment (adohall) [reply app/templates/seller-dashboard.hbs:22]: Ok. We'll have to raise this with Tash. I'll post something to our product chat group.

File: app/templates/seller-account.hbs (hunk 2 above)
Comment (adohall) [reply app/templates/seller-account.hbs:22]: I'll remove this change then.

File: app/templates/seller-dashboard.hbs (hunk 3 above)
Comment (adohall) [reply app/templates/seller-dashboard.hbs:22]: Tash says leave the user email address in, if company name not available.

File: app/templates/seller-account.hbs (hunk 2 above)
Comment (adohall) [reply app/templates/seller-account.hbs]: Removed.

**Pull request merged**